drivers/at86rf2xx/include/at86rf2xx_registers.h

```diff
@@ -361,6 +514,18 @@ extern "C" {
 #define AT86RF2XX_CSMA_SEED_1__CSMA_SEED_1                      (0x07)
 /** @} */
 
+/**
+ * @name   Bitfield definitions for the  TRXPR  Transceiver Pin Register
+ * @{
+ */
+#ifdef MODULE_AT86RFR2
```

Comment:
I'd say wrap also the doxygen in the `#ifdef`.

Reply:
Just for consistency, this is done like 
https://github.com/Josar/RIOT/blob/32b72b4de248b50d314e40b650b8a3f874baa955/drivers/at86rf2xx/include/at86rf2xx_registers.h#L499